float64 thumb
float64 index
float64 middle
float64 ring
float64 pinky
